.eslintrc.js

```diff
@@ -1,35 +1,19 @@
 module.exports = {
-    extends: ['eslint:recommended', 'airbnb-base'],
-    parser: 'babel-eslint',
+    extends: [
+        'eslint:recommended',
+        'plugin:@typescript-eslint/eslint-recommended',
+        'plugin:@typescript-eslint/recommended'
+    ],
+    parser: '@typescript-eslint/parser',
     parserOptions: {
         ecmaVersion: 8,
         sourceType: 'module',
     },
     plugins: [
         'import',
+        '@typescript-eslint'
     ],
-    rules: {
```

Comment:
I used this occasion to simplify the linting rules. If there are opipnions about custom rules I'm happy to add them back but the recommended values are quite a good start.